Lavinia-api/Program.cs

```diff
@@ -18,7 +18,7 @@ public static void Main(string[] args)
                 try
                 {
                     ElectionContext context = services.GetRequiredService<ElectionContext>();
-                    ILogger logger = services.GetRequiredService <ILogger<Program>>();
```

Comment:
This change seems unrelated to the commit message and should have been placed in its own commit.